src/requests.ts

```diff
@@ -55,4 +56,19 @@ function retryRequestInWorker(request: StoredPossumRequest): void {
       removeFailedRequest(request.id);
     }
   };
+
```

Comment:
Since worker pools are going to be the default, why not switch to that mainly instead of being commented.

Reply:
Oh sure, I'm checking a few things out first before I uncomment it.